src/TfvcMigrator/Program.cs

```diff
@@ -117,16 +116,19 @@ private static RootPathChange ParseRootPathChange(string token)
         var unmappedAuthors = changesets.Select(c => c.Author)
             .Concat(changesets.Select(c => c.CheckedInBy))
             .Concat(allLabels.Select(l => l.Owner))
-            .Select(identity => identity.UniqueName)
-            .Where(name => !authorsLookup.ContainsKey(name))
-            .Distinct(StringComparer.OrdinalIgnoreCase)
+            .Where(identity => !authorsLookup.ContainsKey(identity.UniqueName))
+            .GroupBy(i => i.UniqueName)
+            .Select(grouping => grouping.First())
```

Comment:
The OrdinalIgnoreCase comparer should be retained.

Also, because this project targets .NET 6:

```suggestion
            .DistinctBy(i => i.UniqueName, StringComparer.OrdinalIgnoreCase)
```